### PR TITLE
Harden youtube-dl.profile

### PR DESCRIPTION
**Pull request by rusty-snake**

### CODE REVIEW

**Comment by glitsj16** [comment]
@rusty-snake I just noticed you saw my question in https://github.com/netblue30/firejail/commit/1862d24fd9990bfc61f9ae9710f089b3d8038427. Now you're already working on the youtube-dl profile in this PR, if you'd like you can go ahead and integrate the new disable-exec stuff. In the mean time I'm running some tests with the changed you propose here. I happen to have a rather extensive non-default/exotic youtube-dl usage with firejail so give me some time to respond on the results :smile:.

**Comment by glitsj16** [comment]
@rusty-snake Thanks for adding disable-exec. The profile looks really nice now with your extra hardenings. Sadly `memory-deny-write-execute` breaks on Arch Linux for me. So if you would be willing to edit that in this can be merged.

**Comment by glitsj16** [comment]
Great job, thank you! I'll let the travis check do its job and will merge in a minute.

File: etc/youtube-dl.profile [hunk 1]
```diff
@@ -45,8 +52,11 @@ seccomp
 shell none
 tracelog
 
+disable-mnt
```
Comment (SkewedZeppelin) [inline etc/youtube-dl.profile]: media on other drives is common enough use case?
most other media profiles don't have this

Comment (rusty-snake) [reply etc/youtube-dl.profile]: ytdl is only for downloading.

Comment (glitsj16) [reply etc/youtube-dl.profile]: @SkewedZeppelin Nice catch, fixed.

Comment (rusty-snake) [reply etc/youtube-dl.profile]: I still don't see a reason, why ytdl should can access other drivers.

Comment (glitsj16) [reply etc/youtube-dl.profile]: @rusty-snake 
> I still don't see a reason, why ytdl should can access other drivers.

Suppose a user wants to download to a USB-drive mounted at /mnt/my-youtube-usb-stick/foo that will fail with `disable-mnt`. Another reason why @SkewedZeppelin is correct would be that youtube-dl supports `--external-downloader foo`, which also should be able to save straight to a device mounted under /mnt. Or do you have something else in mind?

**Pull request merged**